stablehlo/conversions/tosa/tests/rescale.mlir

```diff
@@ -0,0 +1,136 @@
+// RUN: stablehlo-opt %s --tosa-rescale-legalize-to-stablehlo --split-input-file -verify-each | FileCheck %s
```

Comment:
rename to `legalize_tosa_rescale_to_stablehlo.mlir`

Reply:
done